src/test/java/com/cloudant/tests/DatabaseTest.java

```diff
@@ -71,6 +72,22 @@ public void permissions() {
 
     }
 
+    /**
+     * Test that when called against a DB that is not a Cloudant service
+     * an UnsupportedOperationException is thrown
+     */
+    @Test
+    public void testPermissionsException() {
+        try {
+            Map<String, EnumSet<Permissions>> userPerms = db.getPermissions();
+            //ignore the test in cases where we don't get an exception
+            Assume.assumeTrue(false);
+        }catch(UnsupportedOperationException e){

```

Comment:
Then the test would fail against the Cloudant service because there is no UnsupportedOperationException thrown, but the test wouldn't be skipped because there would be no assume code. It doesn't matter anyway because I can use expected if we skip the test through the extra categories.